README.md

```diff
@@ -1,4 +1,174 @@
-# UVCGAN
+# Overview
 
-This repository will be used to host UVCGAN code.
+This package provides CycleGAN and model implementations used in the `uvcgan`
+paper.
+
+`uvcgan` introduces an improved method to perform an unpaired image-to-image
+style transfer based on a CycleGAN framework. Combined with a new hybrid
+generator architecture UNet-ViT (UNet-Vision Transformer) and a self-supervised
+pre-training, it achieves state-of-the art results on a multitude of style
+transfer benchmarks.
+
+This README file provides brief instructions about how to setup the `uvcgan`
+package and reproduce results of the paper.
+
+The accompanying [Benchmarking](https://github.com/LS4GAN/benchmarking)
+repository contains detailed instructions how competing CycleGAN, CouncilGAN,
+ACL-GAN, and U-GAT-IT models were trained and evaluated.
+
+
```

Comment:
@usert5432  This is very nice document! Let's put two placeholders, one for axiv paper link, and the other for the pre-trained uvcgan model link.

Reply:
Thank you, Ray! I have added these links: https://github.com/LS4GAN/uvcgan/pull/1/commits/34a3880bf7b5f226c98707bbc28f427108e61c54, https://github.com/LS4GAN/uvcgan/pull/1/commits/3e86305d666eb1a1d5519cd6aec1ec157fd1338b